docs/src/pages/components/authenticator/advanced/current-route.angular.mdx

```diff
@@ -26,3 +26,23 @@ You can use `AuthenticatorService` to access `route` string that represents the
   <amplify-authenticator></amplify-authenticator>
 </ng-container>
 ```
+
+#### Authentication Check
+
+If you just need to check if you're authenticated or not, you can use the more straight forward `AuthenticatorService` to access the `authStatus` string. The `authStatus` string can represent the following states:
+
+- `pending`
+- `authenticated`
+- `unauthenticated`
+
+```html
+<!-- Only render this if there's an authenticated user -->
+<ng-container *ngIf="authenticator.authStatus === 'authenticated'">
+  Welcome back!
+</ng-container>
+
+<!-- Render sign-in screen otherwise with authenticator -->
+<ng-container *ngIf="authenticator.authStatus !== 'authenticated'">
+  <amplify-authenticator></amplify-authenticator>
+</ng-container>
```

Comment:
What do you think about adding a loading here?

```suggestion
<!-- Render loading if authStatus is still pending  -->
<ng-container *ngIf="authenticator.authStatus === 'pending'">
  Loading...
</ng-container>

<!-- Only render this if there's an authenticated user -->
<ng-container *ngIf="authenticator.authStatus === 'authenticated'">
  Welcome back!
</ng-container>

<!-- Render sign-in screen otherwise with authenticator -->
<ng-container *ngIf="authenticator.authStatus === 'unauthenticated'">
  <amplify-authenticator></amplify-authenticator>
</ng-container>
```

Reply:
Same question with React below!

Reply:
I can add a loading...

Reply:
ad0eb12cdf3de26d9143b8dd5b2176079db3d237